test/sequential/test-http2-session-timeout.js

```diff
@@ -37,7 +36,7 @@ server.listen(0, common.mustCall(() => {
       const diff = process.hrtime(startTime);
       const milliseconds = (diff[0] * 1e3 + diff[1] / 1e6);
       if (milliseconds < serverTimeout * 2) {
-        setTimeout(makeReq, callTimeout);
+        setImmediate(makeReq);
```

Comment:
This will make it more robust but it won't test the original condition anymore. We need to make sure the combined `setTimeouts` throughout this test run longer than the `serverTimeout`.

Reply:
> We need to make sure the combined `setTimeouts` throughout this test run longer than the `serverTimeout`.

Isn't that what the `if` statement one line above checks?